team/roster_test.go

```diff
@@ -60,10 +61,12 @@ name = "Kiffix"
 [[person]]
   email = "test2@example.com"
   fingerprint = "5C78E71F6FEFB55829654CC5343CC240D350C30C"
+  is_admin = true
```

Comment:
it would be *super* to see a test case where `is_admin` is *not* set (and that it defaults to `false`!!)

Reply:
Well, question, maybe it's not OK to have `is_admin` missing? And we just trash all instances of existing rosters? But in the future we'll have to support adding new fields with graceful defaults.